src/store/index.js

```diff
@@ -1,14 +1,11 @@
 import { createStore } from 'vuex'
-
+import { allCommentsModule } from './allComments'
+import { CurrentUserModule } from './currentUser'
+import { allAccountsModule } from './allAccounts'
 export default createStore({
-  state: {
-  },
-  getters: {
-  },
-  mutations: {
-  },
-  actions: {
-  },
   modules: {
+    allAccounts: allAccountsModule,
+    currentUser: CurrentUserModule,
+    allComments: allCommentsModule
```

Comment:
```suggestion
    allAccounts: allAccountsModule,
    currentUser: currentUserModule,
    allComments: allCommentsModule
```
use consistent case. javascript uses camelCase to name things by default